EasyCommands/CommandParsers/ParameterProcessors.cs

```diff
@@ -211,10 +213,11 @@ public static class ParameterProcessorRegistry {
                 //AmgiguousSelectorPropertyProcessor
                 new BranchingProcessor<SelectorCommandParameter>(
                     TwoValueRule<SelectorCommandParameter,PropertyCommandParameter,DirectionCommandParameter>(
-                        optionalEither<PropertyCommandParameter>(), optionalEither<DirectionCommandParameter>(),
-                        (s,p,d) => new VariableCommandParameter(new AggregatePropertyVariable(PropertyAggregatorType.VALUE, s.Value, p.HasValue() ? p.GetValue().Value : (PropertyType?)null, d.HasValue() ? d.GetValue().Value : (DirectionType?)null))),
+                        requiredEither<PropertyCommandParameter>(), optionalEither<DirectionCommandParameter>(),
```

Comment:
This change enforces Rule 2.  Implicit Aggregate Properties must specify the requested property.